pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -148,6 +161,10 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 	server.CPUCFSQuota = true // enable cpu cfs quota enforcement by default
 	server.MaxPods = 110
 	server.SerializeImagePulls = false // disable serial image pulls by default
+	if enableDNS {
+		// if we are running local DNS, skydns will load the default recursive nameservers for us
+		server.ResolverConfig = ""
```

Comment:
We set clusterDNS to node ip if node DNS is enabled *and* there is no
clusterDNS value from the config.

On May 27, 2016, at 10:31 AM, Jordan Liggitt <notifications@github.com>
wrote:

In pkg/cmd/server/kubernetes/node_config.go
<https://github.com/openshift/origin/pull/7807#discussion_r64913332>:

> @@ -148,6 +161,10 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
>  	server.CPUCFSQuota = true // enable cpu cfs quota enforcement by default
>  	server.MaxPods = 110
>  	server.SerializeImagePulls = false // disable serial image pulls by default
> +	if enableDNS {
> +		// if we are running local DNS, skydns will load the default recursive nameservers for us
> +		server.ResolverConfig = ""

does this affect ClusterDNS?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/7807/files/74f87ae69c2d603cbe84d58b9127f31f0a5620d4#r64913332>,
or mute the thread
<https://github.com/notifications/unsubscribe/ABG_p_fqNfqX3ilQ7nuyI99car8j8A1cks5qFwAwgaJpZM4Hp34y>
.